docs/src/contribute/design/mqtt-topic-design.md

```diff
@@ -4,7 +4,7 @@ tags: [Documentation]
 sidebar_position: 2
 ---
 
-A detailed look into the MQTT topic design giving additional context which the reference page does not provide. It is assumed that you are familiar with the [MQTT Topic Schema](../../references/mqtt_api.md) page, if not, then please read it first before continuing.
+A detailed look into the MQTT topic design giving additional context which the reference page does not provide. It is assumed that you are familiar with the [MQTT Topic Schema](../../references/mqtt-api.md) page, if not, then please read it first before continuing.
```

Comment:
A former commit was doing the reverse.

Question: What's the rational to prefer one over the other?

Reply:
Yes, unfortunately I discovered that the majority of pages was already using `-` instead of `_` too late (I didn't look broad enough...so yes this is a useless commit.